test/e2e/gardener/shoot/create_and_delete_hibernated.go

```diff
@@ -24,27 +24,43 @@ import (
 
 	gardencorev1beta1 "github.com/gardener/gardener/pkg/apis/core/v1beta1"
 	e2e "github.com/gardener/gardener/test/e2e/gardener"
+	"github.com/gardener/gardener/test/framework"
 )
 
 var _ = Describe("Shoot Tests", Label("Shoot", "default"), func() {
-	f := defaultShootCreationFramework()
-	f.Shoot = e2e.DefaultShoot("e2e-hibernated")
-	f.Shoot.Spec.Hibernation = &gardencorev1beta1.Hibernation{
-		Enabled: pointer.Bool(true),
+	var test = func(f *framework.ShootCreationFramework) {
+		f.Shoot.Spec.Hibernation = &gardencorev1beta1.Hibernation{
+			Enabled: pointer.Bool(true),
+		}
+
+		It("Create and Delete Hibernated Shoot", Offset(1), Label("hibernated"), func() {
+			By("Create Shoot")
+			ctx, cancel := context.WithTimeout(parentCtx, 15*time.Minute)
+			defer cancel()
+			Expect(f.CreateShootAndWaitForCreation(ctx, false)).To(Succeed())
+			f.Verify()
+
+			Expect(f.GardenerFramework.VerifyNoRunningPods(ctx, f.Shoot)).To(Succeed())
+
+			By("Delete Shoot")
+			ctx, cancel = context.WithTimeout(parentCtx, 15*time.Minute)
+			defer cancel()
+			Expect(f.DeleteShootAndWaitForDeletion(ctx, f.Shoot)).To(Succeed())
+		})
 	}
 
-	It("Create and Delete Hibernated Shoot", Label("hibernated"), func() {
-		By("Create Shoot")
-		ctx, cancel := context.WithTimeout(parentCtx, 15*time.Minute)
-		defer cancel()
-		Expect(f.CreateShootAndWaitForCreation(ctx, false)).To(Succeed())
-		f.Verify()
+	Context("Shoot with workers", func() {
```

Comment:
I think we should put the workerless tests into a dedicated e2e test flow (and dedicated Prow job). This makes it more clear and simplifies debugging in case something goes wrong.

Reply:
We thought about this, but then we need separate jobs for migration, single-zone, multi-zone too. 
This is unnecessary overhead, doing `kind-up` and `gardener-up` separately.

Reply:
OK, fair enough. Then at lease provide some label ("workerless") so that one can easily include/exclude these tests